tools/xkbcli-how-to-type.1

```diff
@@ -1,44 +1,41 @@
-.TH "XKBCLI\-HOW\-TO\-TYPE" "1" "" "" "libxkbcommon manual"
-.
-.SH "NAME"
-\fBxkbcli\-how\-to\-type\fR \- query how to type a given Unicode codepoint
-.
-.SH "SYNOPSIS"
-\fBxkbcli\fR how\-to\-type [OPTIONS] <codepoint>
-.
-.SH "DESCRIPTION"
-\fBxkbcli how\-to\-type\fR prints key sequences to type the given Unicode codepoint.
-.
-.P
-Pipe into \fBcolumn \-ts $\'\e\et\'\fR for nicely aligned output.
-.
-.SH "OPTIONS"
-.
-.TP
-.B \-\-rules=<rules>
+.Dd July 27, 2020
+.Dt XKBCLI\-HOW\-TO\-TYPE 1
+.Os
+.
+.Sh NAME
+.Nm "xkbcli how\-to\-type"
+.Nd query how to type a given Unicode codepoint
+.
+.Sh SYNOPSIS
+.Nm
+.Op options
+.Ao Ar codepoint Ac
```

Comment:
not sure it's my terminal font, but whatever this uses is a weird one: `⟩` vs the normal `)`. nope, firefox says it's weird too :)

Reply:
It does look weird, removed it now.